packages/styles/code.css

```diff
@@ -0,0 +1,139 @@
+.hljs {
```

Comment:
To avoid conflicts with existing "highlight.js" stylesheets, I would like do something like this:

```ts
// packages/react/src/components/Code/index.tsx
<SyntaxHighlighter {...props} useInlineStyles={false} className="Code">
  // ...
</SyntaxHighlighter>
```

And prefix these classes with:

```css
.Code .hljs {
 /* .. */
}
```